doc/design/volumes/nfsvolumestores.md

```diff
@@ -50,7 +50,7 @@ We want to fail early in the case of issues mounting the volume.  Possible error
 The expectation is the error will be actionable by the user such that if it is a configuration issue related to networking or access, the user can either try the operation again with the right container network configuration, or contact the admin with the action item to allow access to the storage device.
 
 #### VolumeStore
-The `VolumeStore` interface is used by the storage layer to implement the volume storage layer on different backend implementations.  The currently (and only) implementation used by VIC is to manipulate vsphere `.vmdk` backed block devices on the Datastore.  We intend to create a similar implementation for NFSv3.
+The `VolumeStore` interface is used by the storage layer to implement the volume storage layer on different backend implementations.  The currently (and only) implementation used by VIC is to manipulate vsphere `.vmdk` backed block devices on the Datastore.  We have create a similar implementation for `NFS` based volume stores.
```

Comment:
`The currently (and only) implementation used by VIC is to manipulate vsphere `.vmdk` backed block devices on the Datastore.  We have create a similar implementation for `NFS` based volume stores.`  -- might want to re-word since vmdk is no longer the only implementation :)